axiom-cloudwatch-lambda-cloudformation-stack.template.yaml

```diff
@@ -59,7 +59,6 @@ Resources:
     Properties:
       FunctionName: !Ref LambdaFunctionName
       Runtime: python3.9
-      FunctionName: axiom-cloudwatch-lambda
```

Comment:
`FunctionName` is defined twice here, causing `!Ref LambdaFunctionName` command to be overwritten by the hardcoded one